src/common/getType.ts

```diff
@@ -275,8 +275,11 @@ function getOpCodeType(
   program: Program
 ): Type {
   const types = getArgs(expr).map((x) => getType(x, program));
-  function expectVariadicType(expected: Type) {
-    if (types.length < 2 || types.some((x, i) => !isSubtype(x, expected))) {
+  function expectVariadicType(expected: Type, minArity = 2) {
```

Comment:
why default minArity to 2?

Reply:
> why default minArity to 2?

That was the previous behaviour. When I add new parameters to a function I tend to add a default value that keeps the old semantics.